starter-code/app.js

```diff
@@ -53,3 +53,7 @@ app.use(function(err, req, res, next) {
 });
 
 module.exports = app;
+
+app.listen(3000, () => {
```

Comment:
If you were having trouble running npm start, it's because of this this block of code. When you do npm start, you don't need to have app.listen anymore because it runs the server in a different way.